client/src/components/addSourceWizard/addSourceWizard.js

```diff
@@ -43,80 +69,57 @@ class AddSourceWizard extends React.Component {
       });
     };
 
-    Store.dispatch({
-      type: confirmationModalTypes.CONFIRMATION_MODAL_SHOW,
-      title: 'Cancel Add Source',
-      heading: 'Are you sure you want to exit this wizard?',
-      body: 'Exiting this wizard will cancel adding the source.',
-      cancelButtonText: 'No',
-      confirmButtonText: 'Yes',
-      onConfirm: onConfirm
+    stepsConfirmed().then(confirmed => {
```

Comment:
Why do we need to validate the steps on cancel? Maybe only check for no data entered, but should still confirm if I enter an invalid value and accidentally hit cancel.

Reply:
this was left over, along with the majority of the below comments...